### PR TITLE
Patient details after searching on home page

### PR DESCRIPTION
**Pull request by Johngtka**
Features List:
- [x] `added new component for expansion panels with history and patient data`
- [x] `alowed opening new router path with patient/details component`

ezgif com-video-to-gif

### CODE REVIEW

**Comment by Johngtka** [comment]
s&m commit proposal:
`feat: allow opening patient/details after selecting patient on home page (#49)`

File: src/app/empty.state/empty.state.component.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    height: 80vh;
+    height: 66vh;
```
Comment (macrog) [inline src/app/empty.state/empty.state.component.css:5]: why?

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export class HomeComponent {
         });
     }
     openVisitBySearched(patient: Patient) {
```
Comment (macrog) [inline src/app/home/home.component.ts:30]: change the name of the funcion

File: src/app/patient-details/patient-details.component.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<div class="patientDetails">
+    <!-- <mat-expansion-panel>
+        <mat-expansion-panel-header>
+            <mat-panel-title>{{
+                'PATIENT_DETAILS.PATIENT_DATA_PANEL_TITLE' | translate
+            }}</mat-panel-title>
+        </mat-expansion-panel-header>
+
+    </mat-expansion-panel> -->
```
Comment (macrog) [inline src/app/patient-details/patient-details.component.html:9]: remove it !!

**Comment by macrog** [comment]
remove the expandable panel and use just card for patient's history

**Comment by Johngtka** [comment]
> remove the expandable panel and use just card for patient's history

change the history component html or patient/details?

**Comment by macrog** [comment]
@Johngtka is it still in draft?

**Comment by macrog** [comment]
why did you remove the `patient-table.component` in this PR as well? this is done in another PR, there is no need to do it here!?

**Comment by Johngtka** [comment]
> why did you remove the `patient-table.component` in this PR as well? this is done in another PR, there is no need to do it here!?

I just thought there would be collisions at merge

**Comment by macrog** [comment]
> > why did you remove the `patient-table.component` in this PR as well? this is done in another PR, there is no need to do it here!?
> 
> I just thought there would be collisions at merge

you mean conflicts? no there won't be anyconficts, thats the whole point of git that you can have multiple branchas with different changes and you can merge them into master. You only gonna have conflicts if you modify the same parts of code in different branches... which kind of you did now 😄

**Comment by macrog** [comment]
@Johngtka please fix the conflicts!

**Comment by Johngtka** [comment]
> @Johngtka please fix the conflicts!

i.e. I should restore the table component

**Comment by macrog** [comment]
image

what are you doing here? in 1 you have merge master, ok. in 2 you have added patient table? why? it has been removed already and master branch do not have patient-table if you merge this now you will put again patient-table in the master....

**Comment by Johngtka** [comment]
> image
> 
> what are you doing here? in 1 you have merge master, ok. in 2 you have added patient table? why? it has been removed already and master branch do not have patient-table if you merge this now you will put again patient-table in the master....

done

**Pull request merged**